spec/mas/cms/entity/article_spec.rb

```diff
@@ -27,6 +27,7 @@ module Mas::Cms
         slug:        double,
         body:        double,
         non_content_blocks: non_content_blocks,
+        full_path:   double,
```

Comment:
Do we need to add any tests to this, other than adding it into the attributes? 
maybe in this one?https://github.com/moneyadviceservice/mas-cms-client/blob/9ae94834a067fec0f55754ff6b3a0c33173908ac/spec/mas/cms/entity/article_spec.rb#L288

Reply:
@willhall88 What kind of test would you suggest? There is no validation or manipulation of the attribute so any test would seem to be just testing rails.

Reply:
no suggestions, was more checking whether a test needed to be considered or not. happy as is :)